app/bundles/EmailBundle/Controller/EmailController.php

```diff
@@ -1062,12 +1062,6 @@ public function abtestAction($objectId)
 
             $clone = clone $entity;
 
-            //reset
-            $clone->clearStats();
-            $clone->setSentCount(0);
-            $clone->setRevision(0);
-            $clone->setVariantSentCount(0);
-            $clone->setVariantStartDate(null);
             $clone->setIsPublished(false);
```

Comment:
Why are this setters not part of `clone` function?

Reply:
Might not be wanted behavior for all use cases I guess. I just removed those methods that are currently part of `clone`.